tests/test_refObjLoader.py

```diff
@@ -0,0 +1,294 @@
+# This file is part of meas_algorithms.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import os.path
+import tempfile
+import unittest
+
+import numpy as np
+from smatch.matcher import sphdist
+import astropy.time
+
+import lsst.daf.butler
+import lsst.afw.geom as afwGeom
+from lsst.daf.butler import DatasetType, DeferredDatasetHandle
+from lsst.daf.butler.script import ingest_files
+from lsst.meas.algorithms import (ConvertReferenceCatalogTask, ReferenceObjectLoader)
+import lsst.utils
+import lsst.geom
+
+import ingestIndexTestBase
+
+
+class TestRefObjLoader(ingestIndexTestBase.ConvertReferenceCatalogTestBase,
+                       lsst.utils.tests.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
This setUpClass is very long. Is all of this really necessary, and/or could it be split out into free functions?